### PR TITLE
Implement several ballot resolutions

### PR DESCRIPTION
**Pull request by rhdolin**
Changes in this branch: 

Genomic study analysis
* Implement https://jira.hl7.org/browse/FHIR-43744 resolution
* Implement https://jira.hl7.org/browse/FHIR-43745 resolution

Somatic page
* Implement https://jira.hl7.org/browse/FHIR-43574 resolution
* Added gene list to GenomicStudyAnalysis analysisTumorNormalDNA example
* Implement https://jira.hl7.org/browse/FHIR-43517 resolution

Operations page
* Removed STU note

Significance vs. Evidence changes
* Implement https://jira.hl7.org/browse/FHIR-45434 resolution

@khalifa-aly @srikarchamala @bheale @sirkyp

### CODE REVIEW

File: input/fsh/CGGeneral.fsh [hunk 1]
```diff
@@ -47,6 +47,8 @@ One important note is that Annotation is a FHIR data type, this is **NOT** about
 * component ^slicing.discriminator.path = "code"
 * component ^slicing.rules = #open
 * component ^slicing.description = "Slice based on the component.code pattern"
+* component.extension contains RelatedArtifactComponent named related-artifact 0..*
```
Comment (sirkyp) [inline input/fsh/CGGeneral.fsh:50]: @rhdolin - why did you move this to GenomicBase?

File: input/fsh/CGImplications.fsh [hunk 2]
```diff
@@ -30,26 +30,35 @@ Description:    "Properties common to genomic implications expressed as computab
 // * component ^slicing.rules = #open
 // * component ^slicing.description = "Slice based on the component.code pattern"
 * component contains
-    evidence-level 0..*
+    evidence-level 0..* and
+	clinical-significance 0..1
```
Comment (sirkyp) [inline input/fsh/CGImplications.fsh:34]: NIT: looks like formatting is off (line 34 doesn't align with line 33 and others).

File: input/fsh/GGGenomicStudy.fsh [hunk 3]
```diff
@@ -72,23 +72,33 @@ Description: "A genomic study is a set of analyses performed to analyze and gene
...
+
 Extension:   GenomicStudyAnalysisMethodType
 Id:          genomic-study-analysis-method-type
 Title:      "Genomic Study Analysis Method Type"
 Description: "Defines a method type for a genomic analysis"
 * ^context[+].type = #element
 * ^context[=].expression = "Procedure"
 * value[x] only CodeableConcept
-* valueCodeableConcept from GenomicStudyMethodTypeVS (example)
+* valueCodeableConcept from GenomicStudyMethodTypeVS (preferred)
```
Comment (sirkyp) [inline input/fsh/GGGenomicStudy.fsh:91]: Simliar to line 100, should we add some example codes to ^short attribute to this method type?

File: input/fsh/GGGenomicStudy.fsh [hunk 4]
```diff
@@ -113,7 +123,7 @@ Description:   "Genomic Study Analysis Input"
 * extension[file].value[x] only Reference(GenomicDataFile)
 * extension[type] ^short = "GenomicStudy.analysis.input.type"
 * extension[type].value[x] only CodeableConcept
-* extension[type].valueCodeableConcept from GenomicStudyDataFormatVS (example)
+* extension[type].valueCodeableConcept from GenomicStudyDataFormatVS (preferred)
```
Comment (sirkyp) [inline input/fsh/GGGenomicStudy.fsh:126]: Simliar to line 100, should we add some example codes to ^short attribute to this input type?

File: input/fsh/GGGenomicStudy.fsh [hunk 5]
```diff
@@ -130,7 +140,7 @@ Description:   "Genomic Study Analysis Output"
 * extension[file].value[x] only Reference(GenomicDataFile)
 * extension[type] ^short = "GenomicStudy.analysis.output.type"
 * extension[type].value[x] only CodeableConcept
-* extension[type].valueCodeableConcept from GenomicStudyDataFormatVS (example)
+* extension[type].valueCodeableConcept from GenomicStudyDataFormatVS (preferred)
```
Comment (sirkyp) [inline input/fsh/GGGenomicStudy.fsh:143]: Simliar to line 100, should we add some example codes to ^short attribute to this data format?

File: input/fsh/GGGenomicStudy.fsh [hunk 6]
```diff
@@ -249,6 +259,7 @@ Description: "A genomic study analysis is a component of a genomic study."
 * extension contains GenomicStudyAnalysisMethodType named method-type 0..*
                  and GenomicStudyAnalysisChangeType named change-type 0..*
                  and GenomicStudyAnalysisGenomeBuild named genome-build 0..1
+				 and GenomicStudyAnalysisGenomicSourceClass named genomic-source-class 0..1
```
Comment (sirkyp) [inline input/fsh/GGGenomicStudy.fsh:262]: NIT: formatting is off?

File: input/intro-notes/StructureDefinition-therapeutic-implication-intro.xml [hunk 7]
```diff
@@ -1,12 +1,8 @@
...
-  <blockquote class="stu-note">
-    <p>
-    The content in this section may be significantly revised prior to the next ballot after initial use and feedback.
-    </p>
-	  
-  </blockquote>
   <p>
-    This section provides guidance for genomic reporting about the implication of a patient's genetics on the behavior of one or more medications.  This includes making recommendations for medication adjustments.  This portion of the implementation guide relies on the content in the <a href="general.html">General Genomic Reporting</a> and <a href="sequencing.html">Variant Reporting</a> portions of this implementation guide.  Pharmacogenomic reports supplement this information with a set of pharmacogenomic-specific implication profiles. Implementers of pharmacogenomic reporting may also be interested in the <a href="somatics.html">Somatic Reporting</a> section of this implementation guide.
+    This section provides guidance for genomic reporting about the implication of a patient's genetics on the behavior of one or more
```
Comment (sirkyp) [inline input/intro-notes/StructureDefinition-therapeutic-implication-intro.xml:3]: NIT: formatting is off again (it probably renders OK, but does look odd in the source).

File: input/pages/somatics.xml [hunk 8]
```diff
@@ -42,33 +42,33 @@
...
-
-
-Alternatively, look in the table below for just those profiles of interest. </p>
+Alternatively, look in the table below for just those profiles of interest. (The numbers in the 'Label' column correspond to the numbers in the figures up above).</p>
 <table border="1">
 <thead>
   <tr>
   <th>Label</th>
     <th>Resource / Profile</th>
+	<th>Description of somatic usage</th>
```
Comment (sirkyp) [inline input/pages/somatics.xml:51]: NIT: formatting looks off again.

**Review by sirkyp — CHANGES_REQUESTED** [review]

File: input/fsh/CGGeneral.fsh (hunk 1 above)
Comment (rhdolin) [reply input/fsh/CGGeneral.fsh:50]: I'll revert this one.

File: input/fsh/CGImplications.fsh (hunk 2 above)
Comment (rhdolin) [reply input/fsh/CGImplications.fsh:34]: resolved

File: input/fsh/GGGenomicStudy.fsh (hunk 4 above)
Comment (rhdolin) [reply input/fsh/GGGenomicStudy.fsh:126]: added

File: input/fsh/GGGenomicStudy.fsh (hunk 6 above)
Comment (rhdolin) [reply input/fsh/GGGenomicStudy.fsh:262]: fixed

File: input/pages/somatics.xml (hunk 8 above)
Comment (rhdolin) [reply input/pages/somatics.xml:51]: fixed

File: input/fsh/GGGenomicStudy.fsh (hunk 3 above)
Comment (rhdolin) [reply input/fsh/GGGenomicStudy.fsh:91]: added

File: input/intro-notes/StructureDefinition-therapeutic-implication-intro.xml (hunk 7 above)
Comment (rhdolin) [reply input/intro-notes/StructureDefinition-therapeutic-implication-intro.xml:3]: fixed

**Review by sirkyp — APPROVED** [review]
